roles/etcd_migrate/tasks/migrate.yml

```diff
@@ -42,6 +44,7 @@
     --lease-duration 1h
   environment:
     ETCDCTL_API: 3
+    PATH: "/usr/local/bin:/var/usrlocal/bin:{{ ansible_env.PATH }}"
```

Comment:
`/var/usrlocal/bin` should not really be used on Atomic.  It is a detail of how `/usr/local` is made writeable, checked on the last version of RHELAH:

```
# rpm-ostree status
State: idle
Deployments:
● rhel-atomic-host-ostree:rhel-atomic-host/7/x86_64/standard
             Version: 7.3.6 (2017-06-23 16:20:45)
              Commit: e073a47baa605a99632904e4e05692064302afd8769a15290d8ebe8dbfd3c81b

# ls -l /usr/local
lrwxrwxrwx. 3 root root 15 Feb 19  2015 /usr/local -> ../var/usrlocal

# stat /usr/local/bin/oadm
  File: ‘/usr/local/bin/oadm’ -> ‘openshift’
  Size: 9         	Blocks: 0          IO Block: 4096   symbolic link
Device: fd00h/64768d	Inode: 4294213     Links: 1
Access: (0777/lrwxrwxrwx)  Uid: (    0/    root)   Gid: (    0/    root)
Context: unconfined_u:object_r:var_t:s0
Access: 2017-06-29 09:30:24.375797982 +0000
Modify: 2017-06-26 17:33:32.993767546 +0000
Change: 2017-06-26 17:33:32.993767546 +0000
 Birth: -
```